sycl/include/sycl/detail/pi.def

```diff
@@ -126,6 +126,7 @@ _PI_API(piEnqueueMemUnmap)
 _PI_API(piextUSMHostAlloc)
 _PI_API(piextUSMDeviceAlloc)
 _PI_API(piextUSMSharedAlloc)
+_PI_API(piextUSMPitchedAlloc)
```

Comment:
Discussed this internally with the team, but to make the point visible to all reviewers: As this PR changes PI then it will require Unified Runtime changes to match this.

Documentation on how to add experimental features is being written here https://github.com/oneapi-src/unified-runtime/pull/546

You can ask us (@alycm42, @jandres742, @kbenzie) any questions and we'll help you out.

You can also see in-progress work of another experimental change (not even a draft PR yet) here: https://github.com/Bensuo/unified-runtime/pull/1.  The YAML changes are where to look first, a lot of the rest is autogenerated from that.